docs/source/hosts/evernode-host-reputationd.md

```diff
@@ -0,0 +1,50 @@
+# Evernode Reputation for Rewards: Recognizing True Contributors in the Evernode Host Community
+
+The Evernode platform values the commitment and contributions of its hosts. To identify and reward true contributors, Evernode has implemented the "Reputation for Rewards" feature. This article explores the various stages and processes involved in this feature, from opt-in registration to performance checks.
+
+## Opt-In and Registration
+
+The journey begins with the host opting into the reputation assessment process. This step triggers the Evernode reputation hook, which registers the host for the reputation assessment of the upcoming moment. By opting in, hosts voluntarily agree to participate in the assessment, aiming to build and maintain a reputation based on their performance and contributions.
+
+### Setting Up Accounts
+
+Hosts need to maintain a separate Xahau account for Reputation Assessment. During the opt-in process, hosts have the flexibility to either generate a new reputation account or specify an existing one. Similar to funding the host account during installation, hosts must fund this account with deposits in both XAH and EVR:
+
+- XAH covers transaction fees.
+- EVR covers the hourly self-HotPocket instance acquisition for running the Evernode Reputation Contract. The amount required may vary based on the instance's per-moment price.
+
+#### NOTES 
+- Hosts are required to initially deposit adequate XAH funds for setting up their account.
+- Please ensure the secret for this account and the host registration account are backed up to safeguard against any issues with the host machine.
+- When specifying an existing account, it will act as a delegate for the reputation assessment process.
+- If managing `multiple hosts`, a SINGLE reputation account can be used for all, with a separate `DELEGATE HOOK` integrated into the reputation account.
+The host will be responsible for covering the cost of invoking this hook, which triggers upon a specific transaction called `ttACCOUNT_SET`.
+- Furthermore, this setup process asks to reimburse the EVRs received for the host account via reputation instance acquisition. As a result, the EVRs used for reputation assessment purposes will be recycled between the two accounts.
+
```

Comment:
Let's add another point.
- The relationship between the host and reputation address is mapped in WalletLocator. And it's in the following format.
  - <mode (1byte)><mapped-account-id (20bytes)>
  - One to One mapping:
    - Host account: <01><reputation-account-id>
    - Reputation account: <01><host-account-id>
  - One to Many mapping:
    - Host account: <02><reputation-account-id>
    - Reputation account: <02><this-will-be-empty (0's)>